gcp/modules/snapshot-backup/backup-snapshots.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+#mount fast temporal local NVME disk
+mkfs.ext4 /dev/disk/by-id/google-local-nvme-ssd-0
+mkdir /local-ssd
+mount /dev/disk/by-id/google-local-nvme-ssd-0 /local-ssd
+
+# Extract hd images to files
+# GCE connects the disks after the startup script runs
+cd /dev/disk/by-id
+for i in $(ls -1 scsi-0Google_PersistentDisk_pv-database-storage-couchdb-couchdb-* | rev | cut -d'_' -f1 | rev); do
+    dd if=/dev/disk/by-id/scsi-0Google_PersistentDisk_$i | gzip > /local-ssd/$i.gz
```

Comment:
Perhaps you already have plans to change this later in development, but I think it would be better if the data never touches any local disk.

Reply:
+1, or if they do, the disks must be encrypted.